docs/DeveloperGuide.adoc

```diff
@@ -415,6 +415,67 @@ Certain properties of the application can be controlled (e.g App name, logging l
 
 This section describes the details on how the volunteer management features are implemented.
 
+// tag::exportcert[]
+==== Export Volunteer Certificate
```

Comment:
If this feature is not available yet, please add [Coming in V1.3] next to the feature name.